java/code_to_gast/java_expression.py

```diff
@@ -53,6 +54,25 @@ def list_to_attribute_value_node(object_list):
 
 def class_declaration_to_gast(node):
     gast = {"type": "root"}
-    # only support for one function currently
-    gast["body"] = java_router.node_to_gast(node.body[0])
+    '''
+    If the method in class is main function only translate that method and ignore function header
+    We are assuming user only wants body of function translated
+    '''
+    if type(node.body[0]) == javalang.tree.MethodDeclaration and node.body[
+            0].name == "artifical_wrapper_WkVHC":
+        gast["body"] = java_router.node_to_gast(node.body[0].body)
+    else:
+        gast["body"] = java_router.node_to_gast(node.body)
+    return gast
+
+
+def function_delcaration_to_gast(node):
+    gast = {"type": "functionDeclaration"}
+    gast["params"] = java_router.node_to_gast(node.parameters)
+    gast["id"] = {"type": "name", "value": node.name}
+    gast["body"] = java_router.node_to_gast(node.body)
     return gast
+
+
+def formal_parameter_to_gast(node):
```

Comment:
add comment- what is a formal vs. informal parameter? Might not be important but if it isn't then just specify as such

Reply:
+1